File: Gymnasiearbetet-main/project.py
from functions import *


def main():
    global done
    print("Welcome to my password database program!")
    done = True
    account_list = []
    with open("accounts.csv", "r", encoding="utf-8") as file:
        for line in file:
            line = line.strip("\n")
            line_list = line.split(",")
            account_list.append(Accounts(line_list[0], line_list[1], line_list[2]))
    while done == True:
        choice = input("1. Create account \n2. Log in \n3. Exit program\n")
        if choice == "1":
            create_account(account_list)
        elif choice == "2":
            login(account_list)
        elif choice == "3":
            break


main()

File: Gymnasiearbetet-main/se/gymnasiearbetet/projekt/Project.java
package se.gymnasiearbetet.projekt;

import java.io.IOException;
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Project {
    public static void main(String[] args) throws IOException {
        List<Accounts> account_list = new ArrayList<>();
        var addAccounts = Accounts.importAccountsFromFile();
        account_list.addAll((List<Accounts>) addAccounts);
        try (Scanner scanner = new Scanner(System.in)) {
            System.out.println("Welcome to my JAva password database program");
            boolean done = false;
            while (!done) {
                System.out.println("1. Create account \n2. Log in \n3. Exit program");
                var choice = scanner.nextLine();
                if (choice.equals("1")) {
                    Accounts account = Accounts.createAccount(scanner, account_list);
                    account_list.add(account);
                } else if (choice.equals("2")) {
                    var loggedInAccount = Accounts.log_in(scanner, account_list);
                    if (loggedInAccount != null) {
                        Accounts.passwordManagment(scanner, loggedInAccount);
                    }
                } else if (choice.equals("3")) {
                    done = true;
                }
            }
        }
    }
}